src/dhapi/port/lottery_client.py

```diff
@@ -217,6 +219,127 @@ def show_balance(self):
         except Exception:
             raise RuntimeError("❗ 예치금 현황을 조회하지 못했습니다.")
 
+    def show_buy_list(self, json_output=False, start_date=None, end_date=None):
+        try:
+            today = datetime.date.today()
+            
+            if start_date:
+                start_dt = datetime.datetime.strptime(start_date, "%Y%m%d").date()
+            else:
+                start_dt = today - datetime.timedelta(days=14)
+
+            if end_date:
+                end_dt = datetime.datetime.strptime(end_date, "%Y%m%d").date()
+            else:
+                end_dt = today
+
+            data = {
+                "nowPage": 1,
+                "searchStartDate": start_dt.strftime("%Y%m%d"),
+                "searchEndDate": end_dt.strftime("%Y%m%d"),
+                "lottoId": "",
+                "winGrade": 2,
+                "calendarStartDt": start_dt.strftime("%Y-%m-%d"),
+                "calendarEndDt": end_dt.strftime("%Y-%m-%d"),
+                "sortOrder": "DESC",
+            }
+
+            resp = requests.post(self._lotto_buy_list_url, headers=self._headers, data=data, timeout=10)
+            soup = BeautifulSoup(resp.text, "html5lib")
+
+            tables = soup.find_all("table")
+            found_data = []
+            for table in tables:
+                headers = [th.text.strip() for th in table.find_all("th")]
+                rows = []
+                for tr in table.find_all("tr"):
+                    cols = []
+                    detail_info = None
+                    tds = tr.find_all("td")
+                    for i, td in enumerate(tds):
+                        text = td.text.strip()
+                        cols.append(text)
+
+                        if i == 3:
+                            detail_info = self._extract_detail_info(td)
+
+                    if cols:
+                        if detail_info:
+                            numbers = self._get_lotto645_detail(detail_info)
+                            cols[3] = numbers
```

Comment:
```suggestion
                            numbers = self._get_lotto645_detail(detail_info)
                            time.sleep(0.5) # 500ms
                            cols[3] = numbers
```

동행복권 사이트에 부담이 될까봐 조금 걱정이 되었습니다. 이에, 0.5 초 sleep에 대해서 어떻게 생각하세요? 너무 과한 것 같으면 편하게 의견 주세요.